bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/handler/AwattarBridgeHandler.java

```diff
@@ -179,6 +209,29 @@ private void refresh() {
         }
     }
 
+    private Unit<?> getPriceUnit() {
+        Unit<?> priceUnit = UnitUtils.parseUnit("EUR/kWh");
```

Comment:
"€ct" is more common or is this not possible/already possible by the `Unit` class?

Reply:
Not supported by core.